Antidote/CallsManager.m

```diff
@@ -82,14 +83,18 @@ - (void)dealloc
 
 - (void)callToChat:(OCTChat *)chat
 {
+    DDLogVerbose(@"%@, callToChat:%@", self, chat);
+
     if (self.currentCall) {
         NSAssert(NO, @"We should not be able to make another call yet");
         return;
     }
 
-    OCTCall *call = [self.manager callToChat:chat enableAudio:YES enableVideo:NO error:nil];
+    NSError *error;
+    OCTCall *call = [self.manager callToChat:chat enableAudio:YES enableVideo:NO error:&error];
 
     if (! call) {
+        DDLogWarn(@"%@, Error callToChat: %@", self, error.localizedFailureReason);
```

Comment:
I think that logging `error` would give more information than logging just `error.localizedFailureReason`. Can you change it to `error` in all logs or you have any objections?